packages/manager/src/features/Firewalls/FirewallDetail/Rules/firewallRuleEditor.ts

```diff
@@ -246,7 +246,10 @@ export const removeICMPPort = (
   rules: ExtendedFirewallRule[]
 ): ExtendedFirewallRule[] =>
   rules.map((thisRule) => {
-    if (thisRule.protocol === 'ICMP' && thisRule.ports === '') {
+    if (
+      (thisRule.protocol === 'ICMP' || thisRule.protocol === 'IPENCAP') &&
```

Comment:
I could do something like this, but I'm going to leave what I have because it's only two items right now. In the future, a change like this could be made if we add more protocols. 

```suggestion
      ['ICMP', 'IPENCAP'].includes(thisRule.protocol) &&
```